File: dayFour/afterBreak/python/taskTwo.py

sum_total = 0
average = 0
counter = 0
counter = int(counter)

print("Enter your scores below: ")
for scores in range(1, 11):
    score = int(input())
    sum_total = sum_total + score
    counter += 1
average = sum_total/counter
print("The average score is:", average)

File: dayFour/afterBreak/java/TaskTwo.java
import java.util.*;


public class TaskTwo {
	static Scanner input = new Scanner(System.in);
	
	public static void main() {
	
	int sum = 0;
	double averageScore = 0;
	
	System.out.println("Enter the scores below: ");
	int counter= 0;
	
	for(; counter < 10; counter++)
	  {
	    int score = input.nextInt();
	    sum += score;
	  }
	  
	  averageScore = sum/counter;
	  System.out.println(counter);
	System.out.println("Average score is: " + averageScore);
	}
}
